src/main/java/mate/academy/service/UserService.java

```diff
@@ -15,6 +15,13 @@ public class UserService {
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        User user;
+        for (int i = 0; i < users.length; i++) {
+            user = users[i];
```

Comment:
use for-each loop instead
```suggestion
        for (User ...) {
```